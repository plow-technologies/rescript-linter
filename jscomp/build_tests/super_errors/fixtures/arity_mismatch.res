let makeVar = (. ~f, ()) => 34
let makeVariables = makeVar(.~f=f => f)
