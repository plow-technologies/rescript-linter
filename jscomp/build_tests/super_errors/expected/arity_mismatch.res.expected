
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/arity_mismatch.res[0m:[2m2:21-27[0m

  1 [2mâ”‚[0m let makeVar = (. ~f, ()) => 34
  [1;31m2[0m [2mâ”‚[0m let makeVariables = [1;31mmakeVar[0m(.~f=f => f)
  3 [2mâ”‚[0m 

  This uncurried function has type (. ~f: 'a => 'a, unit) => int
  It is applied with [1;31m1[0m arguments but it requires [1;33m2[0m.